sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -303,14 +316,55 @@ public void serialize(ClientSideRequestStatistics statistics, JsonGenerator gene
                 systemInformation.availableMemory = (maxMemory - (totalMemory - freeMemory)) + " KB";
 
                 OperatingSystemMXBean mbean = (com.sun.management.OperatingSystemMXBean)
-                    ManagementFactory.getOperatingSystemMXBean();
-                systemInformation.processCpuLoad = mbean.getProcessCpuLoad()*100 +  " %";
-                systemInformation.systemCpuLoad = mbean.getSystemCpuLoad()*100 +  " %";
+                                                  ManagementFactory.getOperatingSystemMXBean();
+                systemInformation.processCpuLoad = mbean.getProcessCpuLoad() * 100 + " %";
+                systemInformation.systemCpuLoad = mbean.getSystemCpuLoad() * 100 + " %";
                 generator.writeObjectField("systemInformation", systemInformation);
             } catch (Exception e) {
                 // Error while evaluating system information, do nothing
             }
             generator.writeEndObject();
         }
     }
+
+    private class AddressResolutionStatistics {
+        @JsonSerialize(using = ZonedDateTimeSerializer.class)
+        ZonedDateTime startTime;
+        @JsonSerialize(using = ZonedDateTimeSerializer.class)
+        ZonedDateTime endTime;
```

Comment:
Please verify that after moving these field(which doesn't have a explicit serializer)from public to default to after are still coming in diagnostic ,I think we need to do it manually as all field level validation is not their in tests , please refer CosmosResponseDIagnoticsTest file

Reply:
Validated with @simplynaveen20. These look good